Builds/containers/packaging/dpkg/debian/conffiles

```diff
@@ -0,0 +1,4 @@
+opt/ripple/etc/rippled.cfg
+opt/ripple/etc/validators.txt
+opt/ripple/etc/update-rippled-cron
+opt/ripple/bin/update-rippled.sh
```

Comment:
`opt/ripple/etc/update-rippled-cron` and `opt/ripple/bin/update-rippled.sh` should be removed from conffiles.

Reply:
fixed in d930f22f3285cd1cecc6c87059b9a4fd30a0d19f